accounts/views.py

```diff
@@ -80,6 +80,22 @@ def citesphere_token(request):
 		token.save()
 		return Response(status=status.HTTP_201_CREATED)
 
+@api_view(['POST'])
+def check_reset_token(request):
+	username = request.data.get('username', None)
+	if username:
+		try:
+			user = VogonUser.objects.get(username=username)
+		except VogonUser.DoesNotExist:
+			return Response({ "success": False, "message": "User not found" }, status=404)
+	token = request.data.get('token')
+	if token:
+		try:
+			token = ResetToken.objects.get(user=user, token=token)
+		except ResetToken.DoesNotExist:
+			return Response({ "success": False, "message": "Reset token not found" }, status=404)
```

Comment:
use django rest status's. status=status.HTTP_404_NOT_FOUND. This way the implementation is always standardized.